rosie/rosie.d.ts

```diff
@@ -134,11 +134,11 @@ declare namespace rosie {
      *   Factory.define('Person').sequence('id');
      *
      * @param {string} attr
-     * @param {Array.<string>=} dependencies
+     * @param {Array.<string>=} | Function dependencies or builder
      * @param {function(number): *=} builder
      * @return {Factory}
      */
-    sequence(name: string, dependencies?: string[], builder?: Function) : IFactory;
+    sequence(name: string, dependenciesOrBuilder?: Function | string[], builder?: Function) : IFactory;
```

Comment:
True, as does the `attr()` declaration on line 92 (not sure if there are other examples). However, simply adding the two lines as you have would mean only the top one gets the documentation associated with it (at least in VS Code's tooltips). Rather than that, or duplicating the documentation, I have chosen to follow the same choice made elsewhere in this file.